src/components/Widget/GoogleReCaptchaWidget.jsx

```diff
@@ -3,7 +3,7 @@ import { useIntl } from 'react-intl';
 import { Grid } from 'semantic-ui-react';
 import { injectLazyLibs } from '@plone/volto/helpers/Loadable/Loadable';
 
-const GoogleReCaptchaWidget = ({ onVerify, GoogleReCaptcha }) => {
+const GoogleReCaptchaWidget = ({ sitekey, onVerify, GoogleReCaptcha }) => {
   const intl = useIntl();
```

Comment:
In volto we have changed these calls to come from the state and not from the intl object (see https://github.com/plone/volto/issues/3324), so here I would go for something similar to 
```const intl = useSelector((state) => state.intl);```